applications/nova-budapest.md

```diff
@@ -0,0 +1,258 @@
+- **Registered Legal Entity:** Submitted in Google Forms
+
+### Team's experience
+
+Submitted in Google Forms in detail with LinkedIn profiles and relevant experiences.
+
+### Team Code Repos
+
+Submitted in Google Forms.
+
+### Team LinkedIn Profiles (if available)
+
+Submitted in Google Forms.
+
+## Development Status :open_book:
+
+Ongoing. We already submitted the relevant repositories and ready to present the art. No 3D art is ready at the moment.
+
+## Development Roadmap :nut_and_bolt:
+
+### Laying Down the Foundations (Done)
+
+### Creating key and character arts (60% readiness)
+In this step the key arts for the characters will be created as listed in this paragraph. These arts cannot be cheap, and should reflect the highest quality available.
+
+Key arts are one of the main pillars of a successful fundraising as these characters will be minted into NFT’s, and will be the 3D characters in the game. 
+
+### Establishing basic infrastructure (20% readiness)
+In this phase the developers create the project website, decide on the blockchain and NFT technology. We train an AI assistant with the game lore to help newcomers. After they set up the smart contract and the minting and web accounts.
+
+### Pre-development
+The game won’t be able to penetrate to wide audience without any basic gameplay. This needs to be created with 2-3 characters modeled and animated, and some environment pre-modelled, even 4-5 quests playable. Optimally vehicles and VR e-learning demo. Optimal length is 45 seconds of gameplay video.
+
+### Overview
+
+- **Total Estimated Duration:** 8 months
+- **Full-Time Equivalent (FTE):** 5 FTE
+- **Total Costs:** 104,000 EUR
+
+### Milestone 1 — Getting website up and running
+
+- **Estimated duration:** 2 month
+- **FTE:**  3
+- **Costs:** 24,500 USD
+
+The website would contain not only an AI assistant and basic information about the game and the lore, but a complex quest system, where users can play through the website. Completing a certain questline will put the player into a caste, which determines his / her NFT.
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | Pre-payment | OpenAI, Vercel, DigitalOcean |
+| 0b. | Documentation | We will provide only developer-level documentation for the repositories. |
+| 0c. | GitBook | Apart from the developer documentation we should provide another source for future fans to know more about the project. |
+| 0d. | Deployment | We will deploy the stack to Vercel and DigitalOcean|
+| 0e. | Medium | We aim to create a Medium account where we can regularly post updates. |
+| 1. | Node JS Backend | Handles the quest system, achievements, basic user profile, **currently at 20% readiness** |
+| 2. | React Frontend |  **currently at 20% readiness** |
+| 3. | MySQL DB | We would like to store the achievements earned and quest progress and user profile in a classic web2 database. **currently at 20% readiness** |
```

Comment:
The specification of each deliverable should describe the finished state rather than the current state.